idb/postgres/postgres_migrations_integration_test.go

```diff
@@ -7,7 +7,6 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/algorand/go-algorand-sdk/encoding/json"
-
```

Comment:
any particular reason ? we usually group by repo, so the separation makes sense.